src/components/notifications/NotificationsMenu.js

```diff
@@ -0,0 +1,156 @@
+        setAnchorEl();
+    };
+
+    useEffect(() => {
+        const found = notifications?.find(
+            (msg) => msg.message.id === notificationMssg?.message.id
+        );
+        if ((found === null || found === undefined) && notificationMssg) {
+            setNotifications([notificationMssg, ...notifications].slice(0, 10));
+        }
+    }, [notifications, notificationMssg]);
+
+    const { isFetching } = useQuery({
+        queryKey: NOTIFICATIONS_LAST_TEN_KEY,
+        queryFn: getLastTenNotifications,
+        config: {
+            onSuccess: (results) => {
+                setNotifications(results?.messages.reverse());
+                setUnSeenCount(results?.unseen_total);
+            },
+        },
+    });
+
+    return (
+        <Menu
+            anchorEl={anchorEl}
+            id={build(ids.BASE_DEBUG_ID, ids.NOTIFICATIONS_MENU)}
+            open={Boolean(anchorEl)}
+            onClose={handleClose}
+        >
+            <Typography
+                className={classes.header}
+                id={build(
+                    ids.BASE_DEBUG_ID,
+                    ids.NOTIFICATIONS_MENU,
+                    ids.NOTIFICATIONS_HEADER
+                )}
+                variant="h6"
+            >
+                {t("notifications")}
+            </Typography>
+            <Divider />
+
+            {isFetching && <Skeleton variant="rect" width={350} height={400} />}
+            {!isFetching &&
+                notifications.length > 0 &&
+                notifications.map((n, index) => (
+                    <MenuItem
+                        onClick={handleClose}
+                        id={build(ids.BASE_DEBUG_ID, "ids.NOTIFICATION_MENU")}
+                        key={n.message.id}
+                    >
+                        <ListItemText>
+                            <Typography
+                                id={build(
+                                    ids.BASE_DEBUG_ID,
+                                    ids.NOTIFICATIONS_MENU,
+                                    n?.id
+                                )}
+                                variant="subtitle2"
+                            >
+                                {getDisplayMessage(n)}
+                            </Typography>
+                        </ListItemText>
+
+                        <Typography
+                            className={classes.timeStamp}
+                            id={build(
+                                ids.BASE_DEBUG_ID,
+                                ids.NOTIFICATIONS_MENU,
+                                ids.TIME_STAMP
+                            )}
+                            variant="caption"
+                        >
+                            {getTimeStamp(n.message?.timestamp)}
+                        </Typography>
+                    </MenuItem>
+                ))}
+
+            <Divider light />
+            <>
```

Comment:
I see in the browser console that this fragment is causing a bunch of warnings about how the Menu cannot accept a fragment as a child.  I don't think you need to wrap these buttons in anything. I think you can remove this fragment and it should still render fine.